docs/content/dagster-cloud/deployment/agents/running-multiple-agents.mdx

```diff
@@ -7,59 +7,89 @@ platform_type: "cloud"
 
 <Note>This guide is applicable to Dagster Cloud.</Note>
 
-Each Dagster Cloud full deployment (e.g., `prod`) needs to have at least one agent running. Since the agent is lightweight and can tolerate short downtimes such as during an upgrade, a single agent is adequate for most use cases.
+Each Dagster Cloud full deployment (e.g., `prod`) needs to have at least one agent running. A single agent is adequate for many use cases, but you may want to run multiple agents for the following reasons:
 
-If you want to distribute runs across multiple Kubernetes clusters, ECS clusters, EC2 instances, or other environments, you can run multiple agents. For example, one in each cluster, instance, etc.
+- Provide redundancy
```

Comment:
nit grammar here is a little weird.

Maybe "Providing redundancy" instead of "Provide redundancy"

or 

"you may want to run multiple agents in order to:" instead of "you may want to run multiple agents for the following reasons:"